src/main/java/it/pagopa/pagopa/apiconfig/entity/CdiDetailCosmos.java

```diff
@@ -0,0 +1,32 @@
+package it.pagopa.pagopa.apiconfig.entity;
+
+import com.azure.spring.data.cosmos.core.mapping.Container;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder(toBuilder = true)
+@Container(containerName = "cdidetail")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CdiDetailCosmos {
+    private String idBrokerPsp;
+    private String idChannel;
+    private String name;
+    private String description;
+    private Long paymentType;
```

Comment:
are you sure that `paymentType` is a `Long` and not a `String`?

Reply:
I used `Long` because it is the same type on NodoDB, but we can use `String` if it is more human readable

Reply:
fixed